packages/theme/src/components/resizable.ts

```diff
@@ -2,33 +2,33 @@ import type { ComponentMultiStyle } from "@yamada-ui/core"
 import { isGray, transparentizeColor } from "@yamada-ui/utils"
 
 export const Resizable: ComponentMultiStyle<"Resizable"> = {
-  baseStyle: ({ direction: d }) => ({
+  baseStyle: ({ orientation: o }) => ({
```

Comment:
@odendayoko 
Setting direction to orientation here appears to solve the problem.
Would this be inappropriate, as it would revert the previous modification?

```diff
export const Resizable = forwardRef<ResizableProps, "div">(
-  ({ orientation = "horizontal", ...props }, ref) => {
+  ({ direction, orientation = "horizontal", ...props }, ref) => {
    const [styles, mergedProps] = useComponentMultiStyle("Resizable", {
-      orientation,
+      orientation: direction ?? orientation,
      ...props,
    })
    const { className, children, containerRef, ...computedProps } =
      omitThemeProps(mergedProps)
    const { getContainerProps, getGroupProps, ...rest } = useResizable({
      ref,
      ...computedProps,
    })
```